tools/azure-pipelines/jobs-template.yml

```diff
@@ -123,9 +123,9 @@ jobs:
   - script: STAGE=test ${{parameters.environment}} ./tools/azure_controller.sh $(module)
     displayName: Test - $(module)
     env:
-      IT_CASE_S3_BUCKET: $(IT_CASE_S3_BUCKET)
-      IT_CASE_S3_ACCESS_KEY: $(IT_CASE_S3_ACCESS_KEY)
-      IT_CASE_S3_SECRET_KEY: $(IT_CASE_S3_SECRET_KEY)
+      IT_CASE_S3_BUCKET: ${{variables['IT_CASE_S3_BUCKET']}}
```

Comment:
So this was completely broken from the start?

Reply:
Yes  :( I did not test this w/o credentials set.